src/components/Album/Album.js

```diff
@@ -70,46 +73,52 @@ function Album({handleApiError}) {
     return <div>Album not found.</div>;
   }
 
-  const { title, artist, releaseDate, genre, styles, tracklist, coverImg, video} = albumDetails;
+  const { title, artist, releaseDate, genre, styles, tracklist, coverImg, video } = albumDetails;
 
-  if(!isLoading) {
-    return (
-      <div className='album-details'>
-        <div className="album-container">
-          <div className="album-details-container">
-            <h2 className="album-title">{title}</h2>
-            <p className='album-artist'>By: {artist}</p>
-            <img className="cover-image" src={coverImg} alt={`Cover art for ${title}`} />
-            <p>Release Date: {releaseDate}</p>
-            <p>Genre: {genre}</p>
-            {styles && styles.length > 0 && <p>Styles: {styles.join(', ')}</p>}
-          </div>
-        {isUserLoggedIn && 
+  const isAlbumInCollections = currentUser.collections.some(item => item.masterId === masterId);
```

Comment:
curious why you went with a variable instead of state, how is this variable updating?

Reply:
oh, bc a re-render is being done when collections changes?

Reply:
@alivaditis   The reason a variable is used here instead of a state is that this value is derived from the currentUser context and the albumDetails state. It's based on existing data and doesn't need to trigger a re-render or hold any dynamic state by itself.